ssl/internal.h

```diff
@@ -2446,10 +2448,28 @@ struct CERT {
   // pointer to the certificate chain.
   STACK_OF(X509) *x509_chain = nullptr;
```

Comment:
I'm not so clear on why, probably just never got to refactoring it since `chain` is the main certificate chain used for verification. The same question also applies to `x509_leaf`.
We could consider doing a larger refactor on this later.